resources/constants.py

```diff
@@ -933,8 +940,14 @@ def lang_enum(name: str, values: Sequence[str]) -> Dict[str, str]:
     'subtitles.entity.tfc.sabertooth.death': 'Sabertooth dies',
     'subtitles.entity.tfc.sabertooth.attack': 'Sabertooth roars',
     'subtitles.entity.tfc.sabertooth.ambient': 'Sabertooth calls',
-    'subtitles.entity.tfc.sabertooth.hurt': "Sabertooth yowls",
-    'subtitles.entity.tfc.sabertooth.sleep': "Sabertooth snores",
+    'subtitles.entity.tfc.sabertooth.hurt': 'Sabertooth yowls',
+    'subtitles.entity.tfc.sabertooth.sleep': 'Sabertooth snores',
+    'subtitles.entity.tfc.rooster.cry': 'Rooster calls',
+    'subtitles.generic.tfc.dirt_slide': 'Dirt landslides',
```

Comment:
```suggestion
    'subtitles.generic.tfc.dirt_slide': 'Soil landslides',
```
Since it's used for sand/gravel as well.